usaspending_api/references/management/commands/load_gtas.py

```diff
@@ -10,17 +10,19 @@
 logger = logging.getLogger("script")
 
 DERIVED_COLUMNS = {
-    "budget_authority_unobligated_balance_brought_forward_cpe": [1000],
-    "obligations_incurred_total_cpe": [2190],
-    "budget_authority_appropriation_amount_cpe": [1160, 1180, 1260, 1280],
+    "anticipated_prior_year_obligation_recoveries": [1033],
     "borrowing_authority_amount": [1340, 1440],
+    "budget_authority_appropriation_amount_cpe": [1160, 1180, 1260, 1280],
+    "budget_authority_unobligated_balance_brought_forward_cpe": [1000],
     "contract_authority_amount": [1540, 1640],
-    "spending_authority_from_offsetting_collections_amount": [1750, 1850],
-    "other_budgetary_resources_amount_cpe": [1340, 1440, 1540, 1640, 1750, 1850],
-    "obligations_incurred": [2190],
     "deobligations_or_recoveries_or_refunds_from_prior_year_cpe": [1021, 1033],
-    "unobligated_balance_cpe": [2490],
+    "obligations_incurred": [2190],
+    "obligations_incurred_total_cpe": [2190],
```

Comment:
Are `obligations_incurred` and `obligationed_incurred_total_cpe` both suppose to reference `[2190]`?

Reply:
`obligations_incurred` was pre-existing and should be dropped

Reply:
Can we add a TODO or something to notify us in the future that this should be dropped? Would hate for this to be forgotten about.

Reply:
I can file a ticket for this @sethstoudenmier

Reply:
My concern is more with someone trying to use this field that doesn't know a ticket exists, adding more for us to remove in the future. As Tony said it was pre-existing so I won't block on it, but could possibly help to reduce the amount of stuff we might have to remove when we get around to fixing it in the future.

Reply:
I'm not worried since the a simple find-replace will fix that when the field is removed